server/model/monitor.js

```diff
@@ -1259,6 +1290,7 @@ class Monitor extends BeanModel {
                     heartbeatJSON["timezone"] = await UptimeKumaServer.getInstance().getTimezone();
                     heartbeatJSON["timezoneOffset"] = UptimeKumaServer.getInstance().getTimezoneOffset();
                     heartbeatJSON["localDateTime"] = dayjs.utc(heartbeatJSON["time"]).tz(heartbeatJSON["timezone"]).format(SQL_DATETIME_FORMAT);
+                    heartbeatJSON["downTime"] = downTime;
```

Comment:
```suggestion
                    if (downTime !== null) {
                        heartbeatJSON["downTime"] = downTime;
                    }
```

Reply:
Would a simple truthiness check not be fine here?

Reply:
Being able to distinguish between information not available and "0" is a good thing here IMO.

Reply:
Sending the downTime property in heartbeatJSON even when downTime is 0 is a good thing as it tells the notification handler that there is a property called  "downTime" in heartbeatJSON which can be anywhere between 0 - [MAX NUMBER]. Giving the downTime property only when it is larger than 0 will confuse the notification handler where the property sometimes comes and sometimes does not.